routes/home.js

```diff
@@ -0,0 +1,63 @@
+var url = require("url");
+
+module.exports = function(config) {
+  return function(req, res) {
+    var makedetails = "{}";
```

Comment:
I can't do that because when you `decodeURIComponent`, `{ }` will return `[object object]` whereas `"{ }"` will return `"{ }"` which can be JSON parsed.